CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing 
+Thanks for helping us make the HPE Design System better! You came to the right place to start your contribution.
+
+## Project Structure
+The HPE Design System is a monorepo built from two modules.
+See the [README](https://github.com/hpe-design/design-system#welcome-to-hpe-design-system)
+for a break down of the Design System structure.
+
+## How to Contribute 
+Code, Code, Code...
+
+## Submitting Pull Requests 
+To make a pull request you will need a GitHub account. For help, see
+GitHub’s documentation on [forking] and [pull requests].
+1. Fork the design-system repository
+1. Clone the repository
+```
+git clone https://github.com/hpe-design/design-system.git
+```
+3. Install dependencies
+```
+yarn install
+```
+
+### Pre Commits
```

Comment:
```suggestion
4. Implement your changes

### Committing your changes
To commit your changes: `git commit`

Upon running this command, pre-commit checks will run:

- Ensuring consistent code formatting, best practices, and enforce code style guidelines (Prettier & ESLint)
- Executing testing suites:
  - Unit tests (future)
  - End-to-end tests ([TestCafe](https://devexpress.github.io/testcafe/documentation/getting-started/))

#### About TestCafe
TestCafe is tool which assists end-to-end testing (i.e. test behavior and interactions for applications / Websites) across browsers and operating systems. TestCafe executes these tests in live browser instances (as opposed to simulated or headless environments).

NOTE: Things to be aware of while the TestCafe test suites are running.
- TestCafe will launch a browser instance for any browsers you have on your local machine
- Browser windows must be in focus in order for the tests to run. If the browser window is minimized or hidden by other windows, the test will pause and will not complete until the browser is in focus.

```

Reply:
This didn't format like I expected, but hopefully it communicates the gist of what I am after.  Trying to keep this focused on why things are being done and what to expect.